src/footer.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by TTND on 12/27/2016.
+ */
+import React from 'react'
+
+export default class Footer extends React.Component {
+    constructor() {
```

Comment:
constructor is required only if you want to set initial state or initial setup
Not required everytime